src/components/OperatorList.tsx

```diff
@@ -78,13 +78,16 @@ const OperatorList: React.VFC<Props> = React.memo((props) => {
                   op.name
                 )}
               </span>
-              <span className="rarity" title={`Rarity: ${op.rarity + 1} stars`}>
-                <span className="rarity-number">{op.rarity + 1}</span>{" "}
+              <span className="rarity" aria-hidden="true">
+                <span className="rarity-number">{op.rarity + 1}</span>
                 <StarIcon className="rarity-star" />
               </span>
               <span key="opClass" className="operator-class">
                 {operatorClass}
               </span>
```

Comment:
By the same token we should probably do
```html
<span className="operator-class">
  <span className="visually-hidden">Class:</span>{" "}
  {operatorClass}
</span>
```
so as to give a bit more context as to what `{operatorClass}` means.

(I don't know why this `span.operator-class` has a `key` prop–it's unnecessary.)

Reply:
Updated.